app/controllers/runtime/buildpacks_controller.rb

```diff
@@ -6,17 +6,18 @@ def self.dependencies
 
     define_attributes do
       attribute :name, String
+      attribute :stack, String, default: 'unknown'
```

Comment:
After reviewing this PR, I'm curious why we need the `'unknown'` magic string. It seems like we could just use `nil` and display `unknown` in the presentation layer.

Reply:
Also, do we care about the edge case where someone uses a stack named 'unknown'?

Reply:
It seemed clearer to use `unknown` as this makes it clearer that in the 2-phase process of creating the buildpack record and then later uploading the actual bits (and only during this 2nd phase we are able to properly ascertain what the stack really is) that there is an indeterminate state, rather than possibly non-intentionally not initializing it. We also didn't want the column to be optional in the database, and an empty string is just as magical and less obvious.

While the edge case could happen, the chances seem fairly remote. How far do we want to go to prevent something silly like this?